### ISSUE TITLE
Pods project - Update to recommended settings

### ISSUE THREAD

**Issue by sasojadrovski** [CocoaPods/CocoaPods#5042]
* [x] I've read and understood the [*CONTRIBUTING guidelines and have done my best effort to follow](https://github.com/CocoaPods/CocoaPods/blob/master/CONTRIBUTING.md).

# Report

## What did you do?

I have added three custom Schemes to my project.
- ProjectName [Development]
- ProjectName [Staging]
- ProjectName [Production]
Each of these Schemes is linked to a custom Build configuration file. 

I have also added the configurations to the Project's configuration section and have set None configuration set for the "Based on Configuration file", before running pod install.

I run pod install.

## What did you expected to happen?

I expected that all pods will be installed correctly and no warnings will be produced in Xcode.

## What happened instead?

When I opened the workspace, and switched through the different Schemes, I have noticed that for the [Development] scheme (only for this scheme), a warning appears: **Pods.xcodeproj - Update to recommended settings.** If I click on the warning it gives me the following checkboxes (and all of them are checked):

- Project 'Pods' - Turn on "Build Architecture Only" When Debugging
(This will turn on "Build Active Architecture Only" for the "Development" configuration and improve build times by skipping unused architectures.)
- Project 'Pods' - Turn on "Enable Testability" When Debugging
(This will turn on "Enable Testability" for the "Debug" configuration.)
- Project 'Pods' - Turn on "Enable Testability" When Debugging
(This will turn on "Enable Testability" for the "Development" configuration.)
- Project 'Pods' - Turn on "Enable Testability" When Debugging
(This will turn on "Enable Testability" for the "ProjectName [Development]" configuration.)

## Podfile
```
target 'ProjectName' do
pod 'AFNetworking', '~> 3.0'
pod 'CocoaLumberjack', '~> 2.2'
pod 'Crashlytics'
pod 'Fabric'
pod 'SSKeychain', '~> 1.3'
end

target 'ProjectNameTests' do

end

xcodeproj 'ProjectName', {
	'Debug' => :debug,
	'ProjectName [Development]' => :debug, 
	'ProjectName [Staging]' => :debug, 
	'ProjectName [Production]' => :release,
	'Release' => :release
} 
```
Please provide me some info whether this is a known issue when using custom build configurations, or maybe I should configure the Project/Pod file differently.

Thanks in advance!

Cheers,
Sasho

**Comment by benmarten** [comment]
+1

**Comment by segiddins** [comment]
What CocoaPods and Xcode versions are you using?

**Comment by sasojadrovski** [comment]
Hey @segiddins,

Sorry I forgot to mention this... I am using CocaPods 0.39.0 and Xcode 7.3.

If you need some more info, please feel free to ask.

Cheers!

**Comment by segiddins** [comment]
In that case, this has been addressed in 1.0 via updates to the default settings in Xcodeproj.
